test/SILGen/Inputs/usr/include/CoreData.h

```diff
@@ -0,0 +1,12 @@
+@interface NSEntityDescription
```

Comment:
Probably best to import Foundation and inherit from NSObject too. Things unexpectedly being base classes does weird things in Objective-C.